lib/process-services-cloud/src/lib/process/process-filters/components/process-filters-cloud.component.ts

```diff
@@ -43,6 +43,14 @@ export class ProcessFiltersCloudComponent implements OnInit, OnChanges, OnDestro
     @Input()
     showIcons: boolean = false;
 
+    /** (optional) Toggles showing an icon by the side of each filter */
+    @Input()
+    set refreshedFilterKey(value: string[]) {
+        if (value?.length) {
```

Comment:
it's not clear to me what is happening here : We get as input an array. but there is always one element and we delete the set .
1 - I would avoid having operation on the set for an input attributes as this might lead to some hidden bugs. 
2 - If we want a value why don't we just accept a value?

Reply:
@tomaszhanaj This is exactly my point, why instead of passing an array we pass the plain value as it's the thing that we are interested in? we can filter on the parent component to have it as plain string and have angular do the job of triggering the change when the value ACTUAL changes if what i got is correct